opentelemetry-api/src/opentelemetry/util/__init__.py

```diff
@@ -59,8 +67,13 @@ def _load_provider(provider: str) -> Provider:
 
 
 def _load_meter_provider(provider: str) -> "MeterProvider":
```

Comment:
Should we remove the `provider` arg altogether? I don't see when it would be anything different, its hardcoded in both spots:

https://github.com/open-telemetry/opentelemetry-python/blob/cf8b70e77b65686d6bc40db227e4af63ee336cf5/opentelemetry-api/src/opentelemetry/metrics/__init__.py#L608

https://github.com/open-telemetry/opentelemetry-python/blob/18761d78e30a4293abd65a897129b57fbf0c0a15/opentelemetry-api/src/opentelemetry/trace/__init__.py#L457